core/src/main/java/org/apache/carbondata/core/datamap/dev/DataMap.java

```diff
@@ -35,7 +37,8 @@
   /**
    * It is called to load the data map to memory or to initialize it.
    */
-  void init(DataMapModel dataMapModel) throws MemoryException, IOException;
+  void init(DataMapModel dataMapModel, Configuration configuration)
```

Comment:
Add method 'getConfiguration` in DataMapModel

Reply:
done